src/api/cow/index.ts

```diff
@@ -136,12 +136,14 @@ export class CowApi {
 
       if (!response.ok) {
         const errorResponse = await response.json()
-        throw new CowError(errorResponse)
+        throw new OperatorError(errorResponse)
       } else {
         return response.json()
       }
     } catch (error) {
       log.error(logPrefix, 'Error getting trades:', error)
+      if (error instanceof OperatorError) throw error
+
       throw new CowError('Error getting trades: ' + error)
```

Comment:
unrelated to this specific PR, but just noticed that we have a few inconsistencies in how errors are thrown in the api methods. So we have 3 types of errors, CowError, QuoteError and OperatorError. My understanding is that if the api throws a 4xx/5xx the inner throw should reflect that with a OperatorError/QuoteError, and the outer catch should throw a CowError in the case that error is not a OperatorError/QuoteError as we have here.
I'm also not sure if we should also be exporting OperatorError/QuoteError in the root index.ts file apart from CowError, some time ago I remember having issues when checking instances types using `instanceof` for inherited objects. If a consumer receives a OperatorError and they do `error instanceof CowError`, is that true?

Reply:
Yes, that would return `true`. I agree with the fact that we should export `OperatorError` and `QuoteError`